lib/java/src/main/java/org/apache/thrift/transport/THttpClient.java

```diff
@@ -166,6 +167,13 @@ public void setConnectTimeout(int timeout) {
     connectTimeout_ = timeout;
   }
 
+  @Deprecated
+  /**
+   * Use instead {@link
+   * org.apache.hc.client5.http.impl.io.BasicHttpClientConnectionManager#setConnectionConfig} or
+   * {@link
+   * org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager#setDefaultConnectionConfig}
+   */
```

Comment:
comment comes before annotations

Reply:
I modified the order of the comment and annotation